server/lib/nl/detection/utils.py

```diff
@@ -133,18 +134,52 @@ def empty_svs_score_dict():
   return {"SV": [], "CosineScore": [], "SV_to_Sentences": {}, "MultiSV": {}}
 
 
+def empty_var_candidates():
+  return dvars.VarCandidates(svs=[], scores=[], sv2sentences={})
+
+
+# Takes the detected svs and returns
+# 1. sv candidates: svs that are Statistical Variable or Topic
+# 2. non sv candidates: any other detected svs.
+def _get_sv_and_non_sv_candidates(
+    svs_scores_dict: Dict) -> tuple[dvars.VarCandidates, dvars.VarCandidates]:
+  sv_candidates = empty_var_candidates()
+  non_sv_candidates = empty_var_candidates()
+  sv_types = property_values(svs_scores_dict['SV'], 'typeOf')
```

Comment:
Hmmm, given this, can we do this only when it is `allow_entities` ?

Generally, its nice to not do any extra API calls by default unless `bio_ft`

Reply:
done